codebreaker.gemspec

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require_relative 'lib/codebreaker/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'codebreaker'
+  spec.version       = Codebreaker::VERSION
+  spec.authors       = ['Dmitry Latyshev']
+  spec.email         = ['gliffy@ukr.net']
+
+  spec.summary       = 'Codebreaker Game'
+  spec.description   = 'Gem for Codebreaker Game'
+  spec.homepage      = 'https://github.com/Dimasfullxq/codebreaker-gem'
+  spec.license       = 'MIT'
+  spec.required_ruby_version = Gem::Requirement.new('>= 2.3.0')
+
+  spec.metadata['allowed_push_host'] = "TODO: Set to 'http://mygemserver.com'"
+
+  spec.metadata['homepage_uri'] = spec.homepage
+  spec.metadata['source_code_uri'] = 'https://github.com/Dimasfullxq/codebreaker-gem'
+  spec.metadata['changelog_uri'] = 'https://github.com/Dimasfullxq/codebreaker-gem/CHANGELOG.md'
+
+  # Specify which files should be added to the gem when it is released.
+  # The `git ls-files -z` loads the files in the RubyGem that have been added into git.
+  spec.files = Dir.chdir(File.expand_path(__dir__)) do
+    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+  end
+  spec.bindir        = 'exe'
+  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.require_paths = ['lib']
+  spec.add_development_dependency 'rake', '~> 12.3.3'
+  spec.add_development_dependency 'rspec', '~> 3.9'
+  spec.add_development_dependency 'rubocop', '~> 0.86.0'
+  spec.add_development_dependency 'rubocop-performance', '~> 1.6.1'
+  spec.add_development_dependency 'rubocop-rspec', '~> 1.40.0'
+  spec.add_development_dependency 'simplecov', '~> 0.18.5'
+  spec.add_development_dependency 'solargraph', '~> 0.39.10'
```

Comment:
pls use version control with only minor version, not patch

Reply:
done)